src/commands/aksAccount/aksAccount.ts

```diff
@@ -50,18 +49,12 @@ export async function selectSubscriptions(): Promise<void> {
 
     const allSubscriptions = await getSubscriptions(sessionProvider.result, SelectionType.All);
     if (failed(allSubscriptions)) {
-        window.showErrorMessage(allSubscriptions.error);
+        await window.showErrorMessage(allSubscriptions.error);
```

Comment:
💡Side note please: I am cleaning this in your fork as I type, this not need to be `await` here, we can capture and then if there is code failure we return or just return the non-awaited string here, let me make these changes.

Reply:
removed the await in recent commit